WcaOnRails/app/webpacker/components/EditSchedule/EditVenues/index.js

```diff
@@ -33,11 +33,12 @@ function EditVenues({
 
       <Segment basic>
         <Card.Group centered itemsPerRow={2}>
-          {wcifSchedule.venues.map((venue) => (
+          {wcifSchedule.venues.map((venue, index) => (
             <VenuePanel
               key={venue.id}
               venue={venue}
               countryZones={countryZones}
+              sendToFront={index > 0 ? () => dispatch(reorderVenue(index, 0)) : undefined}
```

Comment:
Again, I feel like `VenuePanel` can figure out the dispatch logic by itself and should only retrieve a minimum amount of information to know _if_ it should trigger/show that logic to the user or not.